src/commands/check.ts

```diff
@@ -31,15 +31,16 @@ export async function execute(interaction: CommandInteraction) {
 
     if (response.status === "BLOCKED") {
       await interaction.editReply({
-        content: `🚨 **Alert** 🚨 \n\nThis link is a scam! \`${escapedUrl}\` \n\n_Please **DO NOT** click on this link._`,
+        content: `🚨 **Alert** 🚨 \n\nThis link is a **scam!** \`${escapedUrl}\` \n\n_Please **DO NOT** click on this link._
+        \nFile a dispute [here](https://app.chainpatrol.io/dispute).`,
```

Comment:
I feel like this link is quite hard to find. Could we use a Discord link button instead?

Maybe two buttons:

- "Inspect" -> search page
- "Dispute" -> dispute page